src/test/ui/lint/lint-unused-variables.rs

```diff
@@ -0,0 +1,37 @@
+// Exercise the unused_mut attribute in some positive and negative cases
```

Comment:
This test is about unused variables, not unused mut

Reply:
Oopsss :S